apps/token/src/components/eth-wallet/eth-wallet.tsx

```diff
@@ -163,15 +166,15 @@ const ConnectedKey = () => {
         />
       )}
       <WalletCardActions>
-        <Link className="flex-1" to={`${Routes.STAKING}/associate`}>
-          <span className="flex items-center justify-center w-full text-center px-28 border h-28">
+        <Link className="flex-1 pr-8" to={`${Routes.STAKING}/associate`}>
+          <Button variant={'secondary'} className="w-full">
             {t('associate')}
-          </span>
+          </Button>
         </Link>
-        <Link className="flex-1" to={`${Routes.STAKING}/disassociate`}>
-          <span className="flex items-center justify-center w-full px-28 border h-28">
+        <Link className="flex-1 pl-8" to={`${Routes.STAKING}/disassociate`}>
+          <Button variant={'secondary'} className="w-full">
```

Comment:
chore (blocking): buttons can't be nested within anchor tags

quote: "The [a](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element) element can be wrapped around entire paragraphs, lists, tables, and so forth, even entire sections, so long as there is no interactive content within (e.g., buttons or other links)"

source: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element